CONTRIBUTING.md

```diff
@@ -31,3 +77,32 @@ edtr-io
 
 The tools should be configured in such a way, that you don't need to `build` any packages in development (if you run
 into a problem, please file an issue).
+
+### Dependencies
```

Comment:
@edtr-io/edtr-io-devs (especially @Knorrke @metzgaria): Please peruse that section and keep that in mind for future contributions. Feel free to ask me if you have any questions or if the guideline is not clear. Thanks :)